### PR TITLE
436 display video on avatar

### PR DESCRIPTION
**Pull request by pit2710**
This pull request introduces updates to the video chat functionality by integrating Livekit into the project. The changes include:
- Video Chat Integration:
  - Replaced the existing video chat implementation of FaceCam.cs with Livekit.
  - Transitioned from using RPC (Remote Procedure Calls) to WebRTC for video stream transmission. This change leverages video codecs, improving performance and overall efficiency.
- Media Server Requirements:
  - The new implementation requires a media server, either from Livekit or OpenVidu. Both options support on-premises deployments.
  - For online testing, the free cloud solution from Livekit has been utilized.
- Livekit Authentication:
  - Added a Token server for authentication purposes. The token server is currently provided by me.
- Conference Room Management:
  - The conference room name can be chosen freely. It is crucial to ensure that a room is not used by multiple sessions simultaneously to avoid conflicts and disconnections.

The new implementation maintains all fundamental functionalities of the previous system but with enhanced performance and capabilities due to the use of WebRTC.

### CODE REVIEW

**Comment by koschke** [comment]
Thanks for this PR. Would you be so kind to write a short documentation on how to set up everything in our Wiki at GitHub? This would help me to test your PR and all others to use your new feature. Many thanks!

File: Assets/Scenes/SEENewWorld.unity
Comment (koschke) [inline Assets/Scenes/SEENewWorld.unity:1]: See above. Changes to all Unity assets, in particular the scenes, must be described in detail in a commit message. We need to be able to reproduce a change.

**Review by koschke — CHANGES_REQUESTED** [review]
Thanks for this PR once again. I am looking forward to try it myself.

Before it can be merged my remarks should be addressed.

File: Assets/Resources/Prefabs/Players/CC4/Male1.prefab
[the remark this answers is not in the document]
Comment (pit2710) [reply Assets/Resources/Prefabs/Players/CC4/Male1.prefab:1]: These changes were not intended and have been reversed. I think Unity made the changes automatically.

File: Assets/Plugins/FaceCam/ExampleMaterial.mat
[the remark this answers is not in the document]
Comment (pit2710) [reply Assets/Plugins/FaceCam/ExampleMaterial.mat:1]: These changes were not intended and have been reversed. I think Unity made the changes automatically.

File: Assets/Plugins/ShaderControl/Editor/Resources/BuiltShaders.asset
[the remark this answers is not in the document]
Comment (pit2710) [reply Assets/Plugins/ShaderControl/Editor/Resources/BuiltShaders.asset:1]: These changes were not intended and have been reversed. I think Unity made the changes automatically.

File: Assets/SEE/Tools/VideoChat/LivekitVideo.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+using UnityEngine;
+using Unity.Netcode;
+using SEE.Controls;
+using SEE.Utils;
+
```
[the remark this answers is not in the document]
Comment (pit2710) [reply Assets/SEE/Tools/VideoChat/LivekitVideo.cs:5]: The documentation was added in commit 4fbe0f0.

File: Assets/SEE/Tools/VideoChat/LivekitVideoManager.cs [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+using LiveKit.Proto;
+using RoomOptions = LiveKit.RoomOptions;
+using UnityEngine.UI;
+using TMPro;
+using Unity.Netcode;
+using SEE.Controls;
+
+public class LivekitVideoManager : NetworkBehaviour
+{
+    public string livekitUrl = "ws://localhost:7880";
```
[the remark this answers is not in the document]
Comment (pit2710) [reply Assets/SEE/Tools/VideoChat/LivekitVideoManager.cs:23]: The documentation was added in commit 4fbe0f0.

**Pull request closed without merging**